lib/Target/CBackend/CBackend.cpp

```diff
@@ -2437,6 +2437,7 @@ void CWriter::generateHeader(Module &M) {
   // Support for integers with explicit sizes. This one isn't conditional
   // because virtually all CBE output will use it.
   OutHeaders << "#include <stdint.h>\n"; // Sized integer support
+  OutHeaders << "#include <stdlib.h>\n"; // for abort()
```

Comment:
I think a custom definition that just imports or defines `__builtin_trap` might be more accurate
```c++
extern void abort(void);
#if defined(__GNUC__)\n
extern void __builtin_trap(void);
#else
#define __builtin_trap() abort()
#endif
```
and avoids loading this whole header